### PR TITLE
MF-788 - Remove date and minimize copyright comments

### LINKED ISSUE #788: Remove the year from the license comment
Remove the year from the license comment. This should be done in order to simplify maintenance.

### PR DESCRIPTION
**Pull request by nwneisen**
###  What does this do?
Removes the date from copyright comments to reduce maintenance.  Also minimizes and fixes formatting where needed.

### Which issue(s) does this PR fix/relate to?
Resolves #788 

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
NA

### Did you document any new/modified functionality?
NA

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

**Comment by drasko** [comment]
@nwneisen very nice!

Just for my curiosity - I see that JS files were also fixed. How did you automatize this change? You wrote a script or you used some existing tool?

**Comment by nwneisen** [comment]
@drasko No special tools or scripts.  Most of the changes were done using find and replace with excluded directories in VSCode.  A final search for files containing `Copyright (c)` gave me any special cases that I fixed by hand.

File: load-test/src/test/scala/com/mainflux/loadtest/PublishHttpMessages.scala [hunk 1]
```diff
@@ -1,7 +1,5 @@
 /*
- * Copyright (c) 2018
- * Mainflux
- *
+ * Copyright (c) 2018 Mainflux *
```
Comment (dborovcanin) [inline load-test/src/test/scala/com/mainflux/loadtest/PublishHttpMessages.scala]: Please remove the date from this comment.

File: mkdocs.yml [hunk 2]
```diff
@@ -1,11 +1,9 @@
 #
-# Copyright (c) 2018
-# Mainflux
-#
+# Copyright (c) Mainflux
 # SPDX-License-Identifier: Apache-2.0
 #
 
-copyright: Copyright (c) 2015-2019 Mainflux
```
Comment (dborovcanin) [inline mkdocs.yml:8]: I think that you can leave copyright years here, but I don't mind removing them, as well.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: load-test/src/test/scala/com/mainflux/loadtest/PublishHttpMessages.scala (hunk 1 above)
Comment (nwneisen) [reply load-test/src/test/scala/com/mainflux/loadtest/PublishHttpMessages.scala]: Shoot I missed one.  I had to double check them all.  Looks like this was the only one that slipped by.

File: mkdocs.yml (hunk 2 above)
Comment (nwneisen) [reply mkdocs.yml:8]: I actually debated about this one.  My reasoning was that it was more consistent not to have any copyright dates anywhere.

**Review by drasko — APPROVED** [review]
LGTM

**Review by blokovi — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**